StyledStringChains/NSAttributedString+stylable.swift

```diff
@@ -107,7 +115,19 @@ extension NSMutableAttributedString: Stylable {
 
 infix operator ++
 
-static func ++ (lhs: Stylable, rhs: Stylable) -> Stylable {
+func ++ (lhs: Stylable, rhs: String) -> Stylable {
+    let base = lhs as! NSMutableAttributedString
```

Comment:
force unwrapping on Stylable. As you said, this protocol can be used on any string.
Can you use `lhs.asAttributedString`?

Reply:
It's not working for some reason